app.rb

```diff
@@ -17,7 +17,7 @@
 PURGE_DATABASE = true
 
 DataMapper::Logger.new($stdout, :debug) if settings.environment == :development
-DataMapper::Model.raise_on_save_failure = true 
```

Comment:
I added a script in my editor to auto-strip spaces at end of lines, so hopefully these will stop showing up.